### ISSUE TITLE
crashes in lone echo since update

### ISSUE THREAD

**Issue by sawalrus** [LibreVR/Revive#731]
got a message 'oculus store not found' first time when it was open, second time the revive dashboard crashed

**Comment by sullit** [comment]
yeah this seems to be a persistent issue since 1.1.12 and earlier

**Comment by sawalrus** [comment]
Alright I'll hold tight and hope for a fix. losing too much progress to mid savepoint crashes. Sure hope this wasn't an intentional change on Oculus' part?

**Comment by sawalrus** [comment]
confirming I went back to 1.1.10 and issue is gone. wish I still had that sweet fps upgrade from the later versions tho

**Comment by sullit** [comment]
the LE 3.107 fix has resolved this afaik
